src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1972,18 +1972,104 @@
 
 helps['tag create'] = """
 type: command
-short-summary: Create a tag in the subscription.
+short-summary: Create tags on a specific resource.
+long-summary: >
+    The az tag create cmdlet with an id creates or updates the entire set of tags on a resource, resource group or subscription.
+    This operation allows adding or replacing the entire set of tags on the specified resource, resource group or subscription.
+    The specified entity can have a maximum of 50 tags.
+parameters:
+  - name: --name -n
+    short-summary: The name of the tag to create.
+  - name: --subscription
+    short-summary: Name or ID of subscription. You can configure the default subscription using az account set -s NAME_OR_ID.
+  - name: --resource-id
+    short-summary: The resource identifier for the entity being tagged. A resource, a resource group or a subscription may be tagged.
+  - name: --tags
+    short-summary: The tags to be applied on the resource.
 examples:
   - name: Create a tag in the subscription.
     text: >
         az tag create --name MyTag
+  - name: Creates or updates the entire set of tags on a subscription.
+    text: >
+        az tag create --resource-id /subscriptions/{subId} --tags Dept=Finance Status=Normal
+  - name: Creates or updates the entire set of tags on a resource group.
+    text: >
+        az tag create --resource-id /subscriptions/{sub-id}/resourcegroups/{rg} --tags Dept=Finance Status=Normal
+  - name: Creates or updates the entire set of tags on a resource.
+    text: >
+        az tag create --resource-id /subscriptions/{sub-id}/resourcegroups/{rg}/providers/Microsoft.Compute/virtualMachines/{vmName} --tags Dept=Finance Status=Normal
 """
 
 helps['tag delete'] = """
 type: command
-short-summary: Delete a tag in the subscription.
+short-summary: Delete tags on a specific resource.
+long-summary:
+    The az tag delete cmdlet with an id deletes the entire set of tags on a resource, resource group or subscription.
+parameters:
+  - name: --name -n
+    short-summary: The name of the tag to be deleted.
+  - name: --resource-id
+    short-summary: The resource identifier for the entity being tagged. A resource, a resource group or a subscription may be tagged.
 examples:
   - name: Delete a tag from the subscription.
     text: >
         az tag delete --name MyTag
+  - name: Deletes the entire set of tags on a subscription.
```

Comment:
Deletes-> Delete

Reply:
Have been addressed by the new pr: https://github.com/Azure/azure-cli/pull/14558. Will close this pr as too many conflicts to resolve while merging dev branch. Please kindly do a review for the new pr.